backend/app/models/user.rb

```diff
@@ -12,4 +12,5 @@ class User < ApplicationRecord
   has_many :whitelisted_tokens, dependent: :delete_all
   has_many :blacklisted_tokens, dependent: :delete_all
   has_many :courses, dependent: :delete_all
+  has_many :groups, dependent: :delete_all
```

Comment:
The purpose of this line is to create methods for accessing groups via user objects:
```
User.last.groups
```
But this is not working now
<img width="1961" alt="image" src="https://user-images.githubusercontent.com/5938539/223201849-dccb7d00-1290-4caa-9c15-27da81602db3.png">